docs/installation/setup-gce.rst

```diff
@@ -0,0 +1,31 @@
+.. Single Source Instructions
+
+============================================
+Setting Up Nodes Using Google Compute Engine
+============================================
+
+.. begin-body
+
+You can get a Flocker cluster running using Google Compute Engine.
+You'll need to setup at least two nodes.
+
+#. Create a new cloud server:
+
+   * Visit the `Google Cloud Console <https://console.cloud.google.com/>`_.
+   * Navigate to the project you want to use.
+   * Use the menu in the upper left to navigate to ``Compute Engine``.
+   * Click ``Create Instance``.
+   * Choose a name and a zone for your instance. Zone must be the same for all instances.
+   * Choose a machine type.
+   * For the Boot Disk, choose a supported Linux distribution (either CentOS 7 or Ubuntu 14.04) as your image.
+   * Check ``Allow API access to all Google Cloud services in the same project``.
```

Comment:
I _think_ they changed the look and feel of the instance creation page.  There appears to be an "Identity and API" box where the old trusty checkbox used to be.